### ISSUE TITLE
Won't work on Minecraft 1.21

### ISSUE THREAD

**Issue by giuliotoplad** [Keksuccino/Panoramica#15]
When I try to run this mod with Minecraft 1.21 it crashes.
Could you update Panoramica for Minecraft Fabric 1.21?

**Comment by mpustovoi** [comment]
> When I try to run this mod with Minecraft 1.21 it crashes.

Indeed, because Minecraft 1.21 has changed a lot technically after 1.20.4.

I support the 1.21.X port, since there are simply no analogues to this mod (as far as I know).

**Comment by sumone-lol** [comment]
> I support the 1.21.X port, since there are simply no analogues to this mod (as far as I know).

After searching for a while, I have settled on a decent alternative that provides similar results. [ReplayMod](https://modrinth.com/mod/replaymod)'s screenshot feature includes different projections for use with VR or 3D pictures/videos, one of which is the cubic projection which can be used for Minecraft panoramas. It includes extra steps and is a bit clunky for this purpose, but it works for the most part.
